kotlinx-coroutines-core/common/src/flow/operators/Delay.kt

```diff
@@ -16,7 +16,7 @@ import kotlin.time.*
 
 /**
  * Returns a flow that mirrors the original flow, but filters out values
- * that are followed by the newer values within the given [timeout][timeoutMillis].
+ * that are followed by the newer values within the given [timeoutMillis].
```

Comment:
Please, leave this part of the docs as before (it is consistent with how `sample` is documented, too)